### PR TITLE
Kafkamdm TLS + SASL

### PR DESCRIPTION
**Pull request by jhuggart**
I'm using Confluent Cloud, which requires SASL auth. This PR takes @Dieterbe's changes for TLS, adds SASL options, and alters params to follow naming conventions.

### CODE REVIEW

File: cfg/cfg.go [hunk 1]
```diff
@@ -103,10 +103,13 @@ type Route struct {
 	Topic         string // also used by Google PubSub
 	Codec         string // also used by Google PubSub
 	PartitionBy   string
-	TLSEnabled    bool   `toml:"tls_enabled"`
-	TLSSkipVerify bool   `toml:"tls_skip_verify"`
-	TLSClientCert string `toml:"tls_client_cert"`
-	TLSClientKey  string `toml:"tls_client_key"`
+	TLSEnabled    bool
```
Comment (Dieterbe) [inline cfg/cfg.go:106]: the convention for the toml representation (config file) is to use lower case and underscored.

**Comment by Dieterbe** [comment]
Hi Jake, thanks for the Pr! looks good, but my main objection is the breaking of our toml/config param name conventions. You mention it is to follow "naming conventions", which ones do you mean exactly?

also, is this tested?

**Comment by Dieterbe** [comment]
Oh, I see now what you mean.
our config uses a mixture of conventions... like_this for the majority of config values but camelCase for options within routes. I think this is good then, but still would like to know if you've tested this?

**Comment by cjonesshipt** [comment]
Hey @Dieterbe , @jhuggart @krishnaindani can attest to it being testing inside of the Shipt Org.

**Comment by jhuggart** [comment]
Thanks for the merge @Dieterbe!

**Comment by Dieterbe** [comment]
Thanks to you for the nice contribution :)

**Pull request merged**